scripts/components/nav/bookmarks/bookmark-item.component.js

```diff
@@ -0,0 +1,47 @@
+import stringToHTML from 'utils/stringToHTML';
+import BookmarkTemplate from 'templates/bookmark.handlebars';
+
+export default class {
+  constructor(settings) {
+    this.el= settings.el;
+    this.bookmark = settings.bookmark;
+    this.actions = ['update', 'share-bookmark', 'delete'];
+
+    return this;
+  }
+
+  _setEventListener() {
+    this.bookmarkItem.querySelector('.update').addEventListener('click', () => this._update());
```

Comment:
Well, it's true it makes thing easier but I thought you didn't want to implement it because of some reason (usually performance)  so.. I dealt with it.  Actually, I don't have any preference so.. It's up to you :D